hooks/useCreateMetadata.tsx

```diff
@@ -18,14 +19,16 @@ const useCreateMetadata = () => {
       image: imageUri,
       animation_url: animationUri,
       content: {
-        mime: 'audio/mpeg',
+        mime: animationMimeType ?? mimeType,
```

Comment:
what reason(s) do we have to store BOTH mime types?

Reply:
is it possible to preserve the same functionality without adding an additional state variable?
- if no, what is blocking?